api/bucketsd/service.go

```diff
@@ -50,8 +50,6 @@ import (
 const (
 	// chunkSize for get file requests.
 	chunkSize = 1024 * 32 // 32 KiB
-	// maxArchiveSize is the max bucket size that can be archived to filecoin.
```

Comment:
@sanderpick , I think you created this constant. I realized that there was also some `MaxBucketArchiveSize` (see line 112), but was unused and lacked the wiring to flags/envs.
So I deleted this const, wire things, and let this be configurable via flags.

Reply:
Nice, good call